backend/setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = tracim_backend
-version = 4.0.5 build 067
+version = 4.0.6 build 067
```

Comment:
is the build number correct? @grignards

Reply:
I don't think we plan to create a build tag as the upcoming version will be the 4.1.0.
I merged the two changes to differentiate between `build_67` and this commit.